src/wheatley.ts

```diff
@@ -342,6 +342,50 @@ export class Wheatley {
         labelNames: ["type"],
     });
 
+    critical_error(arg: any) {
+        M.error(arg);
+        send_long_message(this.channels.log, `🛑 Critical error occurred: ${to_string(arg)}`)
+            .catch(() => void 0)
```

Comment:
I think there's no need to .catch here if not doing anything, unless the linter was yelling at you about this

Reply:
I just moved this code over from old global function. If I comment this out, it complains that Promises must be awaited.